### PR TITLE
Add wc command

### LINKED ISSUE #33: wc
I'll be doing wc command

### PR DESCRIPTION
**Pull request by bwzhang96**
Closes #33

### CODE REVIEW

File: _commands/basics/wc.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+---
+
+wc
+-------
+
+Print the number of bytes, words, and lines in files
+
+~~~ bash
+a.txt:
```
Comment (mterwill) [inline _commands/basics/wc.md]: Would like to see this (and similar others in your doc) more like Pat formats it in homework... e.g.

    $ cat a.txt
    hello
    $ wc a.txt
    1

File: _commands/basics/wc.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+
+#### `wc -c a.txt`
+~~~ bash
+$ wc -c a.txt
+71 a.txt
+~~~
+
+##### Break it down
+ * When the -c(equivalent to --bytes) flag is set, wc will output the number of bytes in the given file.
```
Comment (mterwill) [inline _commands/basics/wc.md]: Use Markdown's inline code syntax for these options (surround `-c` and `--bytes` in backticks `).

Please mirror this elsewhere in your documentation as well.

File: _commands/basics/wc.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+~~~ bash
+$ wc -c a.txt
+71 a.txt
+~~~
+
+##### Break it down
+ * When the `-c`(equivalent to `--bytes`) flag is set, wc will output the number of bytes in the given file.
+
+
+####` wc -m a.txt`
```
Comment (the-alex) [inline _commands/basics/wc.md]: The `####` appear in the page, as opposed to making the line an h4 header.

**Comment by bwzhang96** [comment]
I think I updated it to fix the error? For some reason I don't actually see those #### with the old commit on my local page. I just assumed the backtick was in the incorrect location for that command. Please let me know if it's still not fixed.

**Comment by mterwill** [comment]
Looks good to me!

**Pull request merged**